src/test/resources/com/cloudbees/plugins/credentials/impl/validKey.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
whilst the current [PEK key](https://github.com/jenkinsci/credentials-plugin/tree/master/src/test/resources/com/cloudbees/plugins/credentials/impl) is encrypted, would it not be better to create a file with the same key without encyrption in the same place?

Reply:
I don't have the passphrase for the existing key, to create an unencrypted version. Please correct me if I misunderstood your comment.

Reply:
it is in the source code (tests) where it is used -> https://github.com/jenkinsci/credentials-plugin/blob/1e306e8f6c54f8eeef973b3387b8c223bd60ae1c/src/test/java/com/cloudbees/plugins/credentials/impl/CertificateCredentialsImplTest.java#L297-L309

Reply:
Added the FIPS complaint encrypted key and certificate